python/tvm/relax/transform/transform.py

```diff
@@ -681,6 +681,25 @@ def ToMixedPrecision(out_dtype="float32") -> tvm.ir.transform.Pass:
     return _ffi_api.ToMixedPrecision(out_dtype)  # type: ignore
 
 
+def SplitCallTIRByPattern(patterns, fcodegen) -> tvm.ir.transform.Pass:
```

Comment:
looks the transformation is for tir only, it would be good to add it (and `MatchResult`) into folder like python/tvm/tir/transform

Reply:
oh i see, it should be fine in relax transform then. The distinction between the two is becoming blurred.